app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingController.kt

```diff
@@ -33,25 +33,26 @@ interface OnboardingController {
 class DefaultOnboardingController(
     private val onboardingStorage: OnboardingStorage,
     val appStore: AppStore,
-    val context: Context,
+    val activity: Activity,
     val selectedTabId: String?,
 ) : OnboardingController {
 
     override fun handleFinishOnBoarding() {
-        context.settings.isFirstRun = false
+        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
+        activity.settings.isFirstRun = false
         appStore.dispatch(AppAction.FinishFirstRun(selectedTabId))
     }
 
     override fun handleGetStartedButtonClicked() {
-        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.M || Browsers.all(context).isDefaultBrowser) {
+        if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.M || Browsers.all(activity).isDefaultBrowser) {
```

Comment:
nit: Here and below: Having a new private property `context` would allow avoiding these modifications.